sample/RedHatSummit17/sponsors

```diff
@@ -1,53 +1,53 @@
 [{
-		"logo":"/images/sponsors/dell.png",
+		"logo-url":"/images/sponsors/dell.png",
 		"description": "Dell EMC serves a key role in providing the essential infrastructure for organizations to build their digital future, transform IT and protect their most important asset, information. Dell EMC enables our enterprise customers\u2019 IT and digital business transformation through trusted hybrid cloud and big-data solutions, built upon a modern data center infrastructure that incorporates industry-leading converged infrastructure, servers, storage, and cybersecurity technologies.",
 		"level": "7",
 		"url": "http://www.dell.com/redhat",
 		"sponsor_type": "Platinum",
```

Comment:
Please make it `type` refer to https://open-event-api.herokuapp.com/#sponsors. Make this change for all the sponsors.